src/PhpSpec/Console/Formatter.php

```diff
@@ -49,6 +49,7 @@ public function __construct($decorated = false, array $styles = array())
         $this->setStyle('value', new OutputFormatterStyle('yellow'));
         $this->setStyle('lineno', new OutputFormatterStyle(null, 'black'));
         $this->setStyle('code', new OutputFormatterStyle('white'));
+        $this->setStyle('label', new OutputFormatterStyle('white', null, array('bold')));
```

Comment:
I'm wondering if white colors are readable on terminals with light colored background. Or we should not care about this (since there is white colored style for `code` and nobody complained about this)?